webrender_api/src/api.rs

```diff
@@ -171,6 +175,18 @@ impl Transaction {
         self.merge(resources);
     }
 
+    // Note: Gecko uses this to get notified when a transaction that contains
+    // potentially long blob rasterization or scene build is ready to be rendered.
+    // sp that the tab-switching integration can react adequately when tab
+    // switching takes too long. For this use case what matters is that the
+    // notification doesn't fire before scene building.
```

Comment:
it's not clear from the description if it fires after blob rasterization or after the scene building